src/lib/ng-package/package.transform.ts

```diff
@@ -237,11 +237,14 @@ const writeNpmPackage = (pkgUri: string): Transform =>
   pipe(
     switchMap(async graph => {
       const { data } = graph.get(pkgUri);
-      const files = [`${data.src}/LICENSE`, `${data.src}/README.md`];
```

Comment:
@alan-agius4, a few days ago, in https://github.com/ng-packagr/ng-packagr/pull/1840, you've removed `CHANGELOG.md` from the list of files that are automatically copied into the destination. Was the change intentional? Or should I add it back and add a test for it?

Reply:
Since we don’t generate a changelog we can leave it as is unless someone complains.